lib/common/MongoDbLogger.ts

```diff
@@ -0,0 +1,31 @@
+import { LoggerState, MongoClient } from 'mongodb';
+import Logger from '../common/Logger';
+
+/**
+ * Base class that contains the common logging functions for mongodb.
+ */
+export default class MongoDbLogger {
+  static setCommandLogger (client: MongoClient) {
+    client.on('commandSucceeded', (event: any) => {
+      Logger.info(event);
+    });
+    client.on('commandFailed', (event: any) => {
+      Logger.warn(event);
+    });
+  }
+
+  static customLogger = function (_message: string | undefined, state: LoggerState | undefined): void {
```

Comment:
1. Style: use consistent method syntax (like what you have in line 8).
2. Sanity: why do you use not to look message?
3. Style: explicitly define access modifiers (`public`).

```suggestion
  public static customLogger (_message: string | undefined, state: LoggerState | undefined): void {
```

Reply:
Still waiting on the comment regarding `_message`.